src/frontend/src/icp/workers/icrc-wallet.worker.ts

```diff
@@ -50,7 +50,7 @@ const mapTransaction = ({
 	}
 
 	if (nonNullish(ledgerId) && (isTokenCkEthLedger(ledgerId) || isTokenCkErc20Ledger(ledgerId))) {
-		return mapCkEthereumTransaction({ transaction, identity, ...ledgerId });
+		return mapCkEthereumTransaction({ transaction, identity, ...env, ...ledgerId });
```

Comment:
Why this change? Is that related to this PR?

Reply:
Yes, correct! Basically, `env` was not being passed to the function, making it `undefined` and consequently `ckETHExplorerUrl` was `undefined`. 

Do you want me to put it in another PR? I think it is quite related to this one, no?

Reply:
Yes please another PR, thanks!

Reply:
Done in 

https://github.com/dfinity/oisy-wallet/pull/2130